docs_nnx/guides/gemma.ipynb

```diff
@@ -19,16 +19,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Getting Started with Gemma Sampling using NNX: A Step-by-Step Guide\n",
+    "# Get started with Gemma sampling/inference using Flax NNX\n",
     "\n",
-    "You will find in this colab a detailed tutorial explaining how to use NNX to load a Gemma checkpoint and sample from it."
```

Comment:
"guide" in the title, "tutorial" in the first paragraph -> let's use "tutorial".